src/constants/preferences.js

```diff
@@ -67,6 +67,7 @@ export default {
             mentionHighlightBg: '#ffe577',
             mentionHighlightLink: '#166de0',
             codeTheme: 'github',
+            postRethreadHighlight: '#cfebfd',
```

Comment:
I have little experience with new theme preferences. Should we aim to use an existing colour here instead? Do we need a migration strategy otherwise? (Probably questions for you @hmhealey.)

Reply:
Migration isn't a problem any more, but I feel like we should be careful about adding one-off colours like this since it's an extra option in the theme UI specifically for a this feature. Most of the other colours are reused in at least a few places, so it seems odd to have this one very specific one that people may configure

Reply:
I didnt have much experience with the themes and preferences so I just set the values to such that they could be tested and that the themes werent entirely horrible. I didnt spend much time on the feature